### DISCUSSION TITLE
iOS purposeStrings.xml some languages are missing

### DISCUSSION THREAD

**Discussion by pol2095** [airsdk/Adobe-Runtime-Support#2360] [Q&A] [answered]
Hello,

using `purposeStrings.xml`, I noticed that some languages are missing, for example Hungarian (hu),
but it's possible to choose Hungarian in XCode native iOS project.

a discussion [here](https://forums.adobe.com/thread/1405231)

I listed the languages supported by an AIR IPA :
```
cs : Czech
da : Danish
de : German
el : Greek
en : English
es : Spanish
fi : Finnish
fr : French
id : Indonesia
it : Italian
iw : Hebrew
ja : Japanese
ko : Korean
ms : Malay
nb : Norwegian
nl : Dutch
pl : Polish
pt : Portuguese
ru : Russian
sv : Swedish
th : Thai
tr : Turkish
vi : Vietnamese
zh_Hans : Chinese Simplified
zh_Hant : Chinese Traditional
```

I think languages can be added :
```
ar : Arabic
hi : Hindi
hr : Croatian
hu : Hungarian
ro : Romanian
sk : Slovak
uk : Ukrainian
```
Thanks.

**Comment by pol2095** [comment]
Any news about languages missing in *.ipa generated by Air ?

![ipa](https://github.com/airsdk/Adobe-Runtime-Support/assets/6962232/b8bc6755-a1c9-4701-a67b-a1a38fddbf82)

**Reply by ajwfrost** [reply, marked as answer]
I think it's possible for you to add them manually via the supported languages list?
See e.g. https://github.com/airsdk/Adobe-Runtime-Support/issues/444#issuecomment-987262559
But I'm not sure whether we would also perhaps need to have 'default' strings put in?

thanks

**Reply by pol2095** [reply]
Thanks, using `<supportedLanguages>` tag it work.
